CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# Unreleased:
```

Comment:
```suggestion
# Unreleased:
* Replace `ShutdownAlreadyCompleted` with `ShutdownAlreadyTriggered` which contains both the initial shutdown reason, and the second, ignored, shutdown reason.
```

Reply:
But I did forget to list the changes regarding these types.